pos_payment_method_cashdro/static/src/js/screens.js

```diff
@@ -0,0 +1,243 @@
+            url += "&password=" + this.journal.cashdro_password;
+            return url;
+        },
+
+        _cashdro_payment_url: function(parameters) {
+            // Compose the url for a sale report to Cashdro
+            var url = this._cashdro_url()
+            url += "&operation=startOperation&type=4";
+            url += "&posid=pos-" + this.pos.pos_session.name;
+            url += "&posuser=" + this.pos.get_cashier().id;
+            url += "&parameters=" + JSON.stringify(parameters);
+            return url;
+        },
+
+        _cashdro_ack_url: function(operation_id) {
+            // Compose the url for a sale report to Cashdro
+            var url = this._cashdro_url()
+            url += "&operation=acknowledgeOperationId";
+            url += "&operationId=" + operation_id;
+            return url;
+        },
+
+        _cashdro_ask_url: function(operation_id) {
+            // Compose the url for to report a sale to Cashdro
+            var url = this._cashdro_url()
+            url += "&operation=askOperation";
+            url += "&operationId=" + operation_id;
+            return url;
+        },
+
+        _cashdro_finish_url: function(operation_id) {
+            // Compose the url for a sale report to Cashdro
+            var url = this._cashdro_url()
+            url += "&operation=finishOperation&type=2";
+            url += "&operationId=" + operation_id;
+            return url;
+        },
+
+        _cashdro_request: function (url) {
+            // We'll use it for regular requests
+            return $.ajax({
+                url: url,
+                method: "GET",
+                async: true,
+                success: function (response) {
+                    return response;
+                }
+            });
+        },
+
+        _cashdro_request_payment: function (url) {
+            // This is a special request, as we keep requesting the CashDro
+            // until we get the *finished* state that will give us the amount
+            // received in the cashdrawer.
+            var def = $.Deferred();
+            var _request_payment = function(url) {
+              $.ajax({
+                  url: url,
+                  method: "GET",
+                  success: function (response) {
+                      var data = JSON.parse(response.data);
+                      if (data.operation.state === "F") {
+                          def.resolve(response);
+                      } else {
+                          _request_payment(url);
+                      }
+                  }
+              });
+            }
+            _request_payment(url);
+            return def;
+        },
+
+
+        _show_error: function (msg, title) {
+            if (!title) {
+                title =  _t("CashDro Error");
+            }
+            this.pos.gui.show_popup("error", {
+                "title": title,
+                "body": msg,
+            });
+        },
```

Comment:
```suggestion
```

Is this method used?
If yes, maybe remove by simply calling `show_popup`, or at least rename it because `_show_error` is too generic and that's not playing nicely with other modules

Reply:
Sure, they're some missing remanents when I tried to adapt it to a backport of some v13 features. It's to be dropped